src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -47,6 +46,22 @@
 logger = logging.getLogger("pyfluent.launcher")
 
 
+def _remove_unused_args(fluent_launch_mode: LaunchMode = None, **kwargs):
+
+    def _remove_keys(unused_args, **kargs):
+        for arg in unused_args:
+            kargs.pop(arg, None)
+        return kargs
+
+    if fluent_launch_mode == LaunchMode.STANDALONE or LaunchMode.PIM:
+        return _remove_keys(
+            ["start_container", "container_dict", "dry_run", "scheduler_options"],
+            **kwargs,
+        )
+    elif fluent_launch_mode == LaunchMode.CONTAINER:
+        return _remove_keys(["start_container", "scheduler_options"], **kwargs)
+
+
 def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
     """Factory function to create launcher for supported launch modes.
```

Comment:
Is it important  to know that this is a factory function? Descriptions and methods should start with a verb. Use one of these:
Create a launcher for supported launch modes.
Use the factory function to create a launcher for supported launch modes.

Reply:
Done. Thank you.